packages/commercetools/composables/src/getters/userShippingGetters.ts

```diff
@@ -15,16 +15,19 @@ const userGetters: UserShippingGetters<any, any> = {
   getTotal: shipping => shipping.addresses.length,
   getPostCode: address => address ? address.zipCode : '',
   getStreetName: address => address ? address.streetName : '',
+  getStreetNumber: address => address ? address.apartment : '',
```

Comment:
As we have talked about this. Don't we have any dedicated field for that in CT which is not an apartment?